src/Mod/TechDraw/TechDrawTools/CommandPositionSectionView.py

```diff
@@ -49,7 +49,7 @@ def GetResources(self):
         return {'Pixmap': 'TechDraw_ExtensionPositionSectionView.svg',
                 'Accel': "",
                 'MenuText': QT_TRANSLATE_NOOP("TechDraw_PositionSectionView", "Position Section View"),
-                'ToolTip': QT_TRANSLATE_NOOP("TechDraw_PositionSectionView", 
+                'ToolTip': QT_TRANSLATE_NOOP("TechDraw_PositionSectionView",
```

Comment:
Hmm... Would this reset the translation string here ?

Reply:
No, the space comes after the comma and is not part of the string.